burn-import/pytorch-tests/tests/complex_nested/export_weights.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+class ConvBlock(nn.Module):
+    def __init__(self, in_channels, out_channels, kernel_size):
+        super(ConvBlock, self).__init__()
+        # TODO test different groups
```

Comment:
Should this TODO be implemented in this PR?

Reply:
Thanks for catching it. I should complete it. 

Regarding thiserror. I didn't know it didn't work in no-std. I found this crate that we can use: https://crates.io/crates/thiserror-no-std

Reply:
This one is better https://crates.io/crates/snafu, that crate seems a fake copy of the original one. The repository is the same `thiserror` repository

Reply:
OK. Thanks for this one. I will probably use raw Error struct and implement From to handle conversation. I'd like to make no-std compatible.